### ISSUE TITLE
Fireflies connector

### ISSUE THREAD

**Issue by tahpot** [verida/data-connector-server#70]

**Comment by tahpot** [comment]
Need to add a `meeting-transcript` schema.

**Comment by tahpot** [comment]
Fireflies only supports API keys, it doesn't support OAuth connections.

You will need to create a special provider page, much like I did for Telegram (See https://github.com/verida/data-connector-server/tree/main/src/web/provider/telegram).

The Fireflies `connect()` method needs to redirect to the special provider page that you create (`/provider/fireflies`) (See https://https://github.com/verida/data-connector-server/blob/b8d9aa20aec992ca31065ec2065716442544594c/src/providers/telegram/index.ts#L47 for redirect example).

The page can be very simple and just ask for the users API key, directing them with instructions on how to get it.

Once they submit the key, redirect back to the `callback()` endpoint with the user entered API key (`/callback/fireflies?apiKey=${apiKey}`). Your call back method in fireflies can then save the API key as the `accessToken`.

**Comment by chime3** [comment]
Yeah, Telegram also doesn't support oAuth2 authentication, but it provides the SSO throughout `Sign In with Telegram` feature.

I thought that FireFlies might have a way to get apiKey programmatically, like Telegram, and tried to test APIs.

But I couldn't find a solution.

We can contact FireFlies support team at support@fireflies.ai for more information.

### Notes
- The Fireflies team is improving their API to expose more features now, so I might be wrong for relying on the current documentation.
- Very low API rate limit:
  ![Image](https://github.com/user-attachments/assets/e224c427-3083-498e-8d42-0e029f06c59e)

**Comment by chime3** [comment]
### What's been done:

https://github.com/user-attachments/assets/b73cf268-e03c-4262-be71-14c7da42b46b

### Issues:

FireFlies has 4 customer packages: `Free`, `Pro`, `Business`, and `Enterprise`. FireFlies Document says API access is available for all plans
